ow_lander/scripts/arm_find_surface.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+from ow_lander import actions
+from ow_lander import node_helper
+from ow_lander import constants
+
+import argparse
+
+from geometry_msgs.msg import Point, Vector3
+
+parser = argparse.ArgumentParser(
+  formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+  description="Determine the position of a surface by moving the arm's " \
+              "end-effector towards surface until sufficient resistance is " \
+              "encountered.")
```

Comment:
Minor: I'm not sure the backslashes are needed.  I've omitted them in another development with no problems.

Reply:
Looks like I have definitely been using these line splits where they are not needed. They have been removed from this file, but I will save a full purge of the unnecessary ones for later so not to clutter this branch with minor changes. It will probably be done as a commit on noetic-devel.